composer.json

```diff
@@ -42,5 +42,11 @@
         "branch-alias": {
             "dev-master": "0.10.x-dev"
         }
+    },
+    "archive": {
+        "exclude": [
+            "/tests",
+            "/phpunit.xml.dist"
```

Comment:
spec folder and phpspec config file should be here to follow the logic entirely (it's part of the test suite). The .env.dist file is also absolutely required, as well as the make file.